Base/Logic/vtkSlicerApplicationLogic.cxx

```diff
@@ -737,15 +743,46 @@ bool vtkSlicerApplicationLogic::IsPluginInstalled(const std::string& filePath,
     return false;
     }
 
+  // If the plugin is found in the application build directory (or one of its descendent), it is not
+  // considered to be installed. This is generally true unless the plugin is found in the extension
```

Comment:
Do we have somewhere a definition of "installed" and "built-in" module?

Reply:
> not installed = in build tree ?

Since extension can be installed in the build tree in the "special" folder `<applicationHome>/<org-name-or-domain>/Extensions-12345`, saying this is not exactly true.

> not built-in = provided by extension?

Not exactly correct either as module can be specific using option like `--additional-module-path/--additional-module-paths` and in that care they are not installed using the extension manager.

----

I create a "mermaid" diagram to include in our documentation

Reference:
* https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/
* https://sphinxcontrib-mermaid-demo.readthedocs.io/en/latest/

Reply:
I guess what I miss is a very high-level description of what installed and built-in flags are meant to distinguish between.

For example, built-in could refer to "not being manually installed by the user" or "source code is in Slicer source tree". Both could make sense, both could be useful to know.